frontend/tribal-resistance-explorer/style.css

```
@@ -0,0 +1,273 @@
+   Falls back to sensible defaults if global theme variables aren't present. */
+
+#tr-app {
+  --tr-accent: var(--saffron, #ff9933);
+  --tr-green: var(--green, #138808);
+  --tr-gold: var(--primary-gold, #d4af37);
+  --tr-surface: var(--bg-dark-card, rgba(255, 255, 255, 0.06));
+  --tr-border: var(--glass-border, rgba(255, 255, 255, 0.14));
+  --tr-text: var(--text-light, #f8fafc);
+  --tr-muted: var(--text-muted, #94a3b8);
+  --tr-radius: 14px;
+  max-width: 1180px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)